pkgs/by-name/li/littlenavmap/atools.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  libsForQt5,
+}:
+
+stdenv.mkDerivation rec {
```

optional, but preferred
```suggestion
stdenv.mkDerivation (finalAttrs: {
```